### PR TITLE
Move Login Modal styles to separate component

### PR DESCRIPTION
**Pull request by jfurrow**
This separates the login modal styles from the generic modal.

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @pierlo-upitup

File: src/styles/components/login-modal.less [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+    .modal {
+
+      &.login-modal {
+
+        .modal-body {
+          padding-top: @base-spacing-unit-screen-small * 2/3;
+        }
+
+        .login-modal-content {
+          padding-bottom: @modal-body-padding-vertical-screen-small;
```
Comment (pierlo-upitup) [inline src/styles/components/login-modal.less]: why do we use `@modal-body-padding-vertical-screen-small` for `padding-bottom` but `@base-spacing-unit-screen-small` for `-top`?

**Comment by pierlo-upitup** [comment]
I see this 
image

File: src/styles/components/login-modal.less (hunk 1 above)
Comment (jfurrow) [inline src/styles/components/login-modal.less]: Good call, I changed all references to the `modal-body-padding` variables. For some reason, Canvas doesn't provide a `mini` version of this variable, so I removed that media query. Thanks for catching that!

**Comment by jfurrow** [comment]
@pierlo-upitup Should have documented this. You'll see that until the `FormModal` changes get merged in #214.

**Comment by kennyt** [comment]
i still see pierlo's problem and #214 is merged. i also rebased this with master locally

**Comment by jfurrow** [comment]
@kennyt My bad, this PR also needs to merge: https://github.com/mesosphere/dcos-ui-plugins-private/pull/137/files cc @pierlo-upitup

**Comment by kennyt** [comment]
Nice, that PR is merged and the issue is fixed, but this is what it looks like:

px

**Comment by jfurrow** [comment]
@kennyt Good catch! I will adjust the styles to handle wrapping text.

**Comment by jfurrow** [comment]
@kennyt It's fixed now in the plugin's styles, here: https://github.com/mesosphere/dcos-ui-plugins-private/pull/208

**Comment by kennyt** [comment]
@jfurrow is it also possible to set the title of it too so we can hover to read fully?

**Comment by jfurrow** [comment]
@kennyt Good point, done! https://github.com/mesosphere/dcos-ui-plugins-private/pull/209

**Pull request merged**